### DISCUSSION TITLE
What repository to report brew audit issue

### DISCUSSION THREAD

**Discussion by Gcenx** [Homebrew/discussions#1427] [Everyday usage] [answered]
Please see https://github.com/Homebrew/homebrew-core/pull/76696

Before anyone asks yes this build without issue on macOS Catalina/Bug Sur and Apple Silicon devices, macOS Mojave requires Xcode_11.7 that’s been patched using Retroactive.

The problem is brew audit expects `SYMROOT` to be passed when ever using `xcodebuild` when that’s invalid unless passing `-project`.

**Comment by carlocab** [comment, marked as answer]
If the audit is wrong, you can open an issue at Homebrew/brew. Or, better, open a pull request. This is the relevant audit: https://github.com/Homebrew/brew/blob/64b6846d6078525b8569834718f7b093a7dd05e1/Library/Homebrew/rubocops/text.rb#L56-L60

**Reply by Gcenx** [reply]
Thanks, I ended up opening a pull-request to resolve the issue https://github.com/Homebrew/brew/pull/11350
